http.go

```diff
@@ -315,11 +315,15 @@ func attachFile(f *multipart.FileHeader, msg *fbb.Message) error {
 		if converted, err := convertImage(bytes.NewReader(p)); err != nil {
 			log.Printf("Error converting image: %s", err)
 		} else {
-			log.Printf("Done converting '%s'.", f.Filename)
+			if int64(len(converted)) < f.Size {
```

Comment:
That's nested a little deeper than I prefer. How about refactoring so that `func convertImage()` becomes a no-op if the converted image is larger than the original?

Reply:
Done